### PR TITLE
challenge spotify

### PR DESCRIPTION
**Pull request by sergio-arce**

### CODE REVIEW

File: staff/sergio-arce/spotify-template/index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <!-- header -->
+        <header class="header">
+            <img class="header__logo" src="./images/logo-spotify.png" alt="">
+            <button class="header__button">VIEW ON SPOTIFY</button>
+        </header>
+        <!-- hader -->
+  
+        <!-- star -->
+        <section class="start">
+            <h2 class="start__">¡Que empieza el fin de semana!</h2>
```
Comment (lexbarba) [inline staff/sergio-arce/spotify-template/index.html]: @sergi23 this is not a valid class with BEM. Maybe you meant `start__title`?

File: staff/sergio-arce/spotify-template/index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                </div>
+                <div class="container__items">
+                    <img src="./images/rock.jpg" alt="">
+                </div>
+                <div class="container__items">
+                     <img src="./images/img3.jpg" alt="">
+                </div>
+            </div>
+        <footer class="footer">
+            <button class="header__button">VIEW ON SPOTIFY</button>
```
Comment (lexbarba) [inline staff/sergio-arce/spotify-template/index.html]: @sergi23 `header__button`inside `footer`? Please rename them.

File: staff/sergio-arce/spotify-template/index.html (hunk 1 above)
Comment (sergio-arce) [reply staff/sergio-arce/spotify-template/index.html]: donde

File: staff/sergio-arce/spotify-template/index.html (hunk 2 above)
Comment (sergio-arce) [reply staff/sergio-arce/spotify-template/index.html]: done

**Pull request merged**